lang/en.js

```diff
@@ -28,6 +28,9 @@ export default {
 	"components.filter-dimension-set-date-text-value.textHours": "{num, plural, =1 {Last hour} other {Last {num} hours}}",
 	"components.filter-dimension-set-date-text-value.textDays": "{num, plural, =0 {Today} one {Last {num} day} other {Last {num} days}}",
 	"components.filter-dimension-set-date-text-value.textMonths": "Last {num} months",
+	"components.filter-dimension-set-date-time-range-value.descriptiveTextRange": "{startValue} to {endValue}",
```

Comment:
I'll add these lang terms to the other lang files but I am waiting on Jeff for more input there.